src/ValidatableExtensions/ValidatableExtensions.Strings.cs

```diff
@@ -75,6 +75,22 @@ public static partial class ValidatableExtensions
         return ref validatable;
     }
 
+    /// <summary>
+    /// Method overloaded to accept a StringComparison param.
+    /// Throws an exception if the string equals the given <paramref name="otherString"/>.
+    /// For the given <paramref name="comparisonType"/>.
+    /// </summary>
+    /// <remarks>
+    /// The default exception thrown is an <see cref="ArgumentException"/>.
+    /// </remarks>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static ref readonly Validatable<string> IfEquals(this in Validatable<string> validatable, string otherString, StringComparison comparisonType)
+    {
+        Validator.ThrowIfEquals(validatable.Value, validatable.ParamName, validatable.ExceptionCustomizations, otherString, comparisonType);
+
+        return ref validatable;
+    }
+    
```

Comment:
I actually meant we can add another parameter to the existing `IfEquals` which is defaulted to `Ordinal`. Similar to your other PR